### PR TITLE
visidata: 2.9.1 -> 2.10.2

### PR DESCRIPTION
**Pull request by nagy**
Changelog: https://github.com/saulpw/visidata/blob/v2.10.2/CHANGELOG.md

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/visidata/default.nix [hunk 1]
```diff
@@ -20,19 +20,22 @@
 , wcwidth
 , zstandard
 , setuptools
+, importlib-metadata
 , git
 , withPcap ? true, dpkt, dnslib
 , withXclip ? stdenv.isLinux, xclip
+, testers
+, visidata
```
Comment (7c6f434c) [inline pkgs/applications/misc/visidata/default.nix:28]: Is it better to do something with `let self = … ; in self`? This approach seems fragile w.r.t. overrides.

Comment (nagy) [reply pkgs/applications/misc/visidata/default.nix:28]: I was following https://nixos.org/manual/nixpkgs/unstable/#tester-testVersion . Looking at other instances of `testers.testVersion` , I do not see that `let self` construct.

Also, if you want to override this package, you would probably not care about this simplistic test, which just checks for the version output. This is mostly ( i think ) intended to aid automatic updates.

**Comment by markus1189** [comment]
Tested and works fine for me

**Review by markus1189 — APPROVED** [review]

File: pkgs/applications/misc/visidata/default.nix (hunk 1 above)
Comment (7c6f434c) [reply pkgs/applications/misc/visidata/default.nix:28]: Well true that if there are never more tests, this approach is fine…

**Pull request merged**